templates/style.scss

```diff
@@ -140,6 +140,9 @@ div.nav-container {
             display: none;
             @media #{$media-sm} {
                 display: block;
+                border-right: 1px solid #ddd;
```

Comment:
The hardcoded color can be replaced with the already existing $color-border variable.